src/pages/FolderSharedPage.jsx

```diff
@@ -0,0 +1,57 @@
+import { useEffect, useState } from "react";
+import "../styles/FolderSharedPage.css";
+import Footer from "../common/footer/Footer";
+import NavigationBar from "../common/navigationBar/NavigationBar";
+import Cards from "../components/cards/Cards";
+import SearchBar from "../common/searchBar/SearchBar";
+import UserInformation from "../components/folderInformation/FolderInformation";
+import { getSampleCards, getSampleUser } from "../services/api";
+
+function FolderSharedPage() {
+  const [folderOwners, setFolderOwners] = useState([]);
+  const [folderName, setFolderName] = useState();
+  const [SampleCards, setSampleCards] = useState([]);
+  const [sampleUserInfo, setSampleUserInfo] = useState([]);
+
+  // 카드 정보 받아오기
+  const getSampleCardsInfo = async () => {
+    const sampleCardsInfo = await getSampleCards();
+    const folderOwners = sampleCardsInfo.folder.owner;
+    const folderName = sampleCardsInfo.folder.name;
+    const sampleCards = sampleCardsInfo.folder.links;
```

Comment:
```suggestion
   const { owner, name, links } = sampleCardsInfo.folder;
```

구조분해할당으로 조금 더 코드를 간결하게 만들 수 있겠네요 ㅎㅎ